packages/trader/src/App/Components/Layout/Footer/__tests__/toggle-settings.spec.js

```diff
@@ -23,10 +23,11 @@ describe('ToggleSettings', () => {
         const wrapper = shallow(<ToggleSettings />);
         expect(wrapper.contains(<Icon icon='IconSettings' className='footer__icon ic-settings__icon' />)).to.be.true;
     });
-    it('property \'in\' should depend on \'is_settings_visible\'', () => {
-        const wrapper = shallow(<ToggleSettings is_settings_visible={true} />);
-        expect(wrapper.find(Modal).shallow().find(CSSTransition).prop('in')).to.be.true;
-    });
+    // it('property \'in\' should depend on \'is_settings_visible\'', () => {
```

Comment:
Why was this commented out? Please add a comment to the code as to why it was commented out.

Reply:
@mahdipourziaei-binary we test this in modal component